web/l10n/en.json

```diff
@@ -118,6 +118,15 @@
     "eventCommandResult": "Command result",
     "eventGeofenceEnter": "Device has entered geofence",
     "eventGeofenceExit": "Device has exited geofence",
+    "eventAlarm": "Alarms",
+    "alarmSos": "sos",
+    "alarmVibration": "vibration",
+    "alarmMovement": "movement",
+    "alarmOverspeed": "overspeed",
+    "alarmFallDown": "fallDown",
```

Comment:
What is this? It's not a proper English phrase. Is it supposed to be id or displayed value?